torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1754,22 +1754,90 @@ std::tuple<at::Tensor, at::Tensor> FoldConvBatchNorm2dHelper::
   return std::make_tuple(new_w, new_b);
 }
 
+std::vector<c10::optional<IValue>> findEPSWeightBiasConstantsOfBN(
+    const script::Module& bn) {
+  auto graph = bn.get_method("forward").graph();
+  const PatternInfo& pattern_bn = PatternInfo::parse_from_str(R"(
+      graph(%a, %weight, %bias, %running_mean, %running_var,
+          %training, %momentum, %eps, %cudnn_enabled):
+        %bn_out = aten::batch_norm(%a, %weight, %bias, %running_mean,
+            %running_var, %training, %momentum, %eps, %cudnn_enabled)
+        return (%bn_out) )");
+  const Graph& pattern_bn_graph = *pattern_bn.pattern_graph;
+  const auto& bn_vmap = pattern_bn.vmap;
+
+  const auto& matches = findPatternMatches(pattern_bn_graph, *graph);
+  if (matches.size() == 0) {
+    return std::vector<c10::optional<IValue>>(3, c10::optional<IValue>{});
+  }
+  const auto& match = matches[0];
```

Comment:
What happens if there is more than 1 match?

Reply:
Actually this was being handled before where the function would return false. The refactoring to move finding constants to a separate function made that ugly to handle. I think I will just go back to as it was before.